actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -28,6 +28,9 @@ def new_mail
       end
 
       def create_inbound_email(mail)
+        # Bcc header is not encoded by default
+        mail[:bcc].include_in_headers = true if mail[:bcc]
```

Comment:
Since this line isn't in `create_and_extract_message_id!` will that mean, Action Mailbox can't capture any BCC fields in emails?

Reply:
Is there an option we can pass to `Mail.new` in `new_mail` instead? Besides `new_mail` seems like the more suitable place for this.

Reply:
Thanks for taking a look @kaspth!

> Since this line isn't in create_and_extract_message_id! will that mean Action Mailbox can't capture any BCC fields in emails?

The only reason this is a problem in these two places is because a `Mail` object is initialized and then serialized with `to_s` before passing it along, that doesn't occur with the other services afaik since the raw data forwarded by the other service is stored without creating a `Mail` object first (I only tested with Mailgun which included the `Bcc` header correctly).

> Is there an option we can pass to Mail.new in new_mail instead? Besides new_mail seems like the more suitable place for this.

The only reference I found to `include_in_headers` is within the `Bcc` field itself so I don't think it can be passed through `Mail.new`. Putting this in `new_mail` does seem suitable, I'll update the PR with that change.